From: Santiago Vila <sanvila@debian.org>
Date: Mon, 25 Nov 2024 21:50:00 +0100
Subject: Disable flaky tests

Bug-Debian: https://bugs.debian.org/1017144
---
 tests/CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index ba96bf4..ad3f498 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -187,19 +187,6 @@ if(NOT GAMMARAY_CLIENT_ONLY_BUILD)
             ${injectorType}
             ${PROJECT_BINARY_DIR}/${BIN_INSTALL_DIR}/connectiontest
         )
-        add_test(
-            connectiontest-${injectorType}-filter
-            ${PROJECT_BINARY_DIR}/${BIN_INSTALL_DIR}/gammaray
-            --inprocess
-            -unittest
-            --listen
-            tcp://127.0.0.1
-            -filtertest
-            -i
-            ${injectorType}
-            ${PROJECT_BINARY_DIR}/${BIN_INSTALL_DIR}/connectiontest
-        )
-        set_tests_properties(connectiontest-${injectorType}-filter PROPERTIES ENVIRONMENT GAMMARAY_TEST_FILTER=1)
     endforeach()
 endif()
 
